src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1107,6 +1107,14 @@ describe('ReactDOMComponent', function() {
         'Warning: This browser doesn\'t support the `onScroll` event'
       );
     });
+
+    it('should not warn when server-side rendering `onScroll`', function() {
+      spyOn(console, 'error');
+      ReactDOMServer.renderToString(React.createElement('div', {
```

Comment:
Let's just use JSX here. We should also make sure this test fails without your code change. I don't think it will - we'd need to do to the same mocking so isEventSupported is false.